src/test/kotlin/com/igorwojda/integer/countupanddown/challenge.kt

```diff
@@ -4,7 +4,17 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countUpAndDown(n: Int): List<Int> {
-    TODO("not implemented")
+    val myArrayList = arrayListOf<Int>()
```

Comment:
Simplify this. Use ranges, and collectors/copying (https://kotlinlang.org/docs/reference/collection-aggregate.html), and plus operator (https://kotlinlang.org/docs/reference/collection-plus-minus.html). Can actually be an easy to read one-liner.